examples/python/interactions_anova.py

```diff
@@ -1,7 +1,7 @@
 
 ## Interactions and ANOVA
 
-# Note: This script is based heavily on Jonathan Taylor's class notes http://www.stanford.edu/class/stats191/interactions.html
+# Note: This script is based heavily on Jonathan Taylor's class notes http://web.stanford.edu/class/stats191/notebooks/Interactions.html
```

Comment:
can you make this "https"?  (i just verified this works).  Other than that, LGTM, @josef-pkt